index.js

```diff
@@ -210,7 +210,8 @@ const mercurius = fp(async function (app, opts) {
       entityResolversFactory: undefined,
       subscriptionContextFn,
       keepAlive,
-      fullWsTransport
+      fullWsTransport,
+      additionalRouteProps: opts.additionalRouteProps
```

Comment:
Hello! Worth adding some docs and types for this do you think?

Reply:
Yes indeed